client/app/dedicated/server/ipmi/dedicated-server-ipmi.html

```diff
@@ -1,212 +1,227 @@
-<div class="row"
-     data-ng-controller="ImpiCtrl"
-     data-ng-init="init()">
+<div
+    class="container-fluid mb-5 px-0"
```

Comment:
I'm curious about why you are applying `px-0` class to the `.container-fluid` element.

Does the container is helpful in your case?

Reply:
Its not required. I have removed it